src/main/java/seedu/address/logic/commands/AssetCommand.java

```diff
@@ -16,9 +17,9 @@
 /**
  * Edits the details of an asset in the address book.
  */
-public class EditAssetCommand extends Command {
+public class AssetCommand extends Command {
 
-    public static final String COMMAND_WORD = "edita";
+    public static final String COMMAND_WORD = "asset";
     public static final Prefix PREFIX_OLD = new Prefix("old/");
     public static final Prefix PREFIX_NEW = new Prefix("new/");
```

Comment:
Should we change `/` to `\`? Similarly, should `old/` be changed to `o\` to follow the single letter conventions of other commands?

Reply:
Need the input of @bryanyee33 here.

Reply:
Update: changed to `o\`